src/PopEngine/index.ts

```diff
@@ -147,8 +127,9 @@ export class PopEngine {
     let isAlreadyShowing = this.isPopAlreadyOpenForGroup(groupId);
 
     // clear any timeouts and do a timeout and show pop
-    this.clearTimeout(targetElement);
-    this._timeouts.hoverdelay = setTimeout(function(): void {
+    timeoutManager._maybeClearTimeout(timeoutManager._timeouts.timeToHoverOnPop, groupId);
+    timeoutManager._maybeClearTimeout(timeoutManager._timeouts.hoverdelay, null);
```

Comment:
What happens when groupId is nul in this case? This is used in several cases, but I'm not 100% sure what this does.